tagging/apps/write_manifest.py

```diff
@@ -24,10 +22,12 @@
 MARKDOWN_LINE_BREAK = "<br />"
 
 
-def write_build_history_line(
-    config: Config, filename: str, all_tags: list[str]
-) -> None:
-    LOGGER.info("Appending build history line")
+def get_build_history_line(config: Config, filename: str, container: Container) -> str:
```

Comment:
The idea is that get-like functions are more testable.
And we don't mix writing to files and creating file content